Uchu.World/Handlers/WorldInitializationHandler.cs

```diff
@@ -191,13 +194,15 @@ private async Task SendCharacterXmlDataToClient(Character character, IRakConnect
             var xml = new byte[bytes.Length - 3];
 
             Buffer.BlockCopy(bytes, 3, xml, 0, bytes.Length - 3);
-
+            
+            var template = gmlevel >= 2 && gmlevel < 5 ? 10103 : 1;
```

Comment:
seeing the enum doesn't include a level with 3 or 4 as value, why not write this as
`var template = gmlevel == GameMasterLevel.MythranWithGlow ? 10103 : 1`?

Reply:
oh yeah

Reply:
Good catch, I like this solution :)

Reply:
Missing a semicolon after this line now